modules/flake/overlays.nix

```diff
@@ -3,29 +3,20 @@
   # Provide overlay to add `nix-snapshotter`.
   flake.overlays.default = self: super:
     let
-      nix-snapshotter = self.callPackage ../../package.nix {};
-
-      containerd = super.containerd.overrideAttrs(o: rec {
-        version = "1.7.14";
-        src = self.fetchFromGitHub {
-          owner = "containerd";
-          repo = "containerd";
-          rev = "v${version}";
-          hash = "sha256-okTz2UCF5LxOdtLDBy1pN2to6WHi+I0jtR67sn7Qrbk=";
-        };
-        patches = (o.patches or []) ++ [
-          # See: https://github.com/containerd/containerd/pull/9864
-          ./patches/containerd-import-compressed.patch
```

Comment:
I think we still need to patch `containerd` here unless I'm mistaken. Although the PR was merged I don't think the [changes](https://github.com/containerd/containerd/pull/9864/commits/34c545824f5d49b771c13cc8e311bb3bec315fc0) have made their way into the version of [containerd currently in 24.05](https://github.com/NixOS/nixpkgs/blob/a781ff33ae258bbcfd4ed6e673860c3e923bf2cc/pkgs/applications/virtualization/containerd/default.nix#L14).

Reply:
Ah yes! My bad, no worries then.